docker-compose.yml

```diff
@@ -78,6 +78,11 @@ services:
       - AZURE_STORAGE_ACCOUNT_KEY=${AZURE_STORAGE_ACCOUNT_KEY}
       - AZURE_STORAGE_CONTAINER=${AZURE_STORAGE_CONTAINER}
       - PUBLIC_URL_PREFIX=${PUBLIC_URL_PREFIX}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
```

Comment:
Can you move the three AWS speech/translation credentials up to above STORAGE_TYPE=${STORAGE_TYPE} line?